### PR TITLE
Lxqt2 labwc session

### PR DESCRIPTION
**Pull request by stefonarch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,35 +50,43 @@ cmake ..  -DCMAKE_INSTALL_PREFIX=/usr  -DCMAKE_BUILD_TYPE=Debug && make -j4
...
   will be imported or set at first run.See
-  `~/.config/lxqt-wayland/labwc/environment` and the "libinput" section in `rc.xml`.
-* Mouse cursor and size are synced and can be set using "Appearance" settings, session restart required. GTK settings have to be updated after changes.
-* Global shortcuts are handled exclusively in `~/.config/lxqt-wayland/labwc/rc.xml`.
-* Refer also to this file for some lxqt-settings (e.g. positions)
-* Some panel plugins do not work on wayland, see [lxqt-panel] for more information.
+  `~/.config/labwc/environment` and the "libinput" section in `rc.xml`.
+  
+* Mouse cursor and size are synced and can be set using "Appearance" settings,
+session restart required. GTK settings have to be updated after changes.
```
Comment (Consolatis) [inline README.md]: On latest master the session restart should not be required anymore, on 0.7.1 it is required.

File: config/labwc/rc.xml [hunk 2]
```diff
@@ -72,36 +72,12 @@
...
-    Other positions are not recommended -->
-    <windowRule identifier="lxqt-panel" skipTaskbar="yes" skipWindowSwitcher="yes" MatchOnce="true" fixedPosition="yes">
-      <action name="ToggleAlwaysOnBottom"/>
-      <action name="MoveTo" x="0" y="0"/>
-      <action name="ToggleKeybinds"/>
-    </windowRule>
     <windowRule identifier="lxqt-runner" skipTaskbar="yes" skipWindowSwitcher="yes">
       <action name="ToggleAlwaysOnTop"/>
       <action name="SnapToRegion" region="runner"/>
     </windowRule>
```
Comment (Consolatis) [inline config/labwc/rc.xml]: You seem to have removed the `runner` region but this rule still uses it.

File: README.md (hunk 1 above)
Comment (stefonarch) [reply README.md]: Thanks, I forgot. But `labwc -r` is needed, no?

File: config/labwc/rc.xml (hunk 2 above)
Comment (stefonarch) [reply config/labwc/rc.xml]: It copies atm the default labwc settings to `~/.config/`  if no config is present, but I've to change that again and ship this labwc folder as default starts mako and waybar. The rule isn't needed anymore, all components are working with layer-shell now. Only taskbar is the missing piece, when we have on I'll remove also waybar/sfwbar/yatbfw.

Settingsexporter isn't working too somehow on fresh accounts I noticed yesterday. Thanks for reviewing!

File: README.md (hunk 1 above)
Comment (Consolatis) [reply README.md]: > Thanks, I forgot. But `labwc -r` is needed, no?

yes (same for keyboard layout)

Comment (stefonarch) [reply README.md]: Done, thanks.

File: config/labwc/rc.xml (hunk 2 above)
Comment (stefonarch) [reply config/labwc/rc.xml]: Should work now. Today LXQt 2 should be merged in git, except wayland-taskbar PR for the panel.

**Comment by stefonarch** [comment]
Atm there is the panel and waybar both at bottom, default panel config contains also showdesktop which crashes the panel (on kwin it works). To fix later.

**Pull request merged**